aten/src/THS/generic/THSTensor.cpp

```diff
@@ -283,6 +304,27 @@ THSTensor *THSTensor_(newTranspose)(THSTensor *self, int d1, int d2) {
  * reshaping methods
  ******************************************************************************/
 
+/*** Helper methods ***/
+THLongTensor *THSTensor_(toCSR)(int64_t const *indices, int64_t dim, int64_t nnz) {
```

Comment:
Calling the size of a dimension `dim` is extremely confusing. We usually use this name for an argument that should range from 0 to number of dimensions in a tensor

Reply:
ok, I will rename it.

Reply:
Can you please still rename it? Thanks!